<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>363</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_titulo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Alterar usuário e senha</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_novo_usuario">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nova_senha">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_usuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuário</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_senha">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Senha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 172, 114);</string>
   </property>
   <property name="text">
    <string>voltar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salvar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 172, 114);</string>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
